crates/aptos/src/genesis/config.rs

```diff
@@ -35,11 +36,20 @@ pub struct Layout {
     pub users: Vec<String>,
     /// ChainId for the target network
     pub chain_id: ChainId,
+    /// Whether to allow validators to join post genesis
+    #[serde(default)]
+    pub allow_new_validators: bool,
+    /// Initial lockup period for genesis validators
+    #[serde(default)]
```

Comment:
`initial_lockup_period_duration_secs` defaults to 0.  Do these defaults make sense? What if people set `allow_new_validators` to be true and leave `initial_lockup_period_duration_secs` to its default?

Should we have better defaults these three fields?

Reply:
+1. I think we also should have min lockup/max lockup and min/max stake set and reward rates in layout as well. These will be directly passed into the Stake module. See https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/configs/Stake.move#L33 and https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/vm-genesis/src/lib.rs#L195